rskj-core/src/test/java/org/ethereum/rpc/Web3ImplTest.java

```diff
@@ -2393,6 +2399,38 @@ void callWithoutReturn() {
         Assertions.assertEquals("0x", result);
     }
 
+    @Test
+    void whenEthGetFilterChanges_throwFilterNotFoundException() {
+        RskSystemProperties config = mock(RskSystemProperties.class);
+
+        when(config.isMinerClientEnabled()).thenReturn(false);
+        when(config.minerClientAutoMine()).thenReturn(false);
+
+        Web3RskImpl web3 = (Web3RskImpl) createWeb3();
+
+        Assertions.assertThrowsExactly(
+                RskJsonRpcRequestException.class,
+                () -> web3.eth_getFilterChanges("0x01"),
+                "filter not found"
+        );
+    }
+
+    @Test
+    void whenEthGetFilterLogs_throwFilterNotFoundException() {
+        RskSystemProperties config = mock(RskSystemProperties.class);
+
+        when(config.isMinerClientEnabled()).thenReturn(false);
+        when(config.minerClientAutoMine()).thenReturn(false);
+
+        Web3RskImpl web3 = (Web3RskImpl) createWeb3();
+
+        Assertions.assertThrowsExactly(
```

Comment:
as mentioned above, check the error code.

Reply:
Done